src/cljs/rems/application.cljs

```diff
@@ -424,85 +429,109 @@
       (rf/dispatch [::set-field id (.-name filecontent)])
       (rf/dispatch [::save-attachment id form-data description]))))
 
+(defn- readonly-field [{:keys [id value]}]
+  [:div.form-control {:id id} (str/trim (str value))])
+
+(defn- format-diff [diff]
+  (map (fn [[change text]] (cond
+                             (pos? change) [:ins text]
+                             (neg? change) [:del text]
+                             :else text))
+       diff))
+
+(defn- diff-field [{:keys [id value previous-value]}]
+  (let [dmp (js/diff_match_patch.)
+        diff (.diff_main dmp (str/trim (str previous-value)) (str/trim (str value)))]
```

Comment:
[minor] could you extract this stuff into something like `(formatted-diff value previous-value)`